### DISCUSSION TITLE
Explanation on transmitting data inside flows

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#19471] [Feature Requests]
### Summary

In this simple example
![image](https://github.com/directus/directus/assets/13746133/86502f15-044f-4cee-b213-1da1796979a6)
how to have in the Send mail all the values of the updated item?
Currently the {{$trigger.payload}} only shows the modified fields.

### Basic Example

--

### Motivation

Better flows creator documentation or videos.

### Detailed Design

--

### Must Have:
- A
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

--

### Alternatives

--

### Adoption Strategy

--

**Comment by Pachat** [comment]
Thanks to the excellent Bryant's video https://youtu.be/k6XfpMHX8xA?t=580 I guess that I have to add a `Read Data` operation after the Condition to read the item been updated. The problem is that the `IDs` field, where I put `{{$trigger.keys[0]}}` is emptied on Save. 
So I don't know how to load the other fields of the row and not only the modified field (Status).

**Comment by Pachat** [comment]
The solution is
- Add a Read Data between the `Condition` and `Send Email`
- Choose the collection `Support` and set the IDs to `"{{$trigger.keys[0]}}"` (be aware of #19474)
- Use `{{$last}}` in the mail body

Not easy to find when creating your first flow. 
This very common use case could be an example flow in the doc.

![Flow Send Mail With Content-2](https://github.com/directus/directus/assets/13746133/a4e494f1-0b79-43bd-97c7-903fce47c026)

**Comment by Pachat** [comment]
And another Read Data to retrieve user's created mail address 


![Flow Send Mail With Content-3](https://github.com/directus/directus/assets/13746133/9cfe1501-02bd-4f7a-ba61-606a6449377a)

**Comment by rijkvanzanten** [comment]
Closing this as it's not a feature request.

If you think you've ran into an issue, please [open a Bug Report](https://github.com/directus/directus/issues/new?assignees=&labels=&projects=&template=bug_report.yml). If you have a usage question, [Discord](https://directus.chat) is the platform we use for that 👍🏻

**Reply by Pachat** [reply]
I agree. It's more a question of usage.
I got it working, and indeed, flows are great!
My request was more oriented towards documentation.
They deserve a few concrete examples to save time.
